runtime/src/lib.rs

```diff
@@ -268,7 +197,7 @@ where
 /// write the impl yourself).
 ///
 /// If you don't need notifications, use `dispatch`.
```

Comment:
Comment bump?

Reply:
it would be nice if this maybe documented how to implement `NotificationHandler` in a way that ignores notifications...

Reply:
Fixed out-of-date comment (good catch, thank you), replaced it with a pointer to the NotificationHandler trait, which now has a full example of how to implement it in a don't-care fashion -- as well as an explanation of why the methods aren't defaulted.